.github/workflows/checksrc.yml

```diff
@@ -41,23 +41,35 @@ jobs:
       - name: check
         run: git ls-files -z "*.[ch]" | xargs -0 -n1 ./scripts/checksrc.pl
 
-  codespell-cmakelint:
-    runs-on: ubuntu-latest
+  codespell-cmake-python:
```

Comment:
To me, using `cmakelint` in the name seems to tell us more relevant info about the job than `cmake-python` does...